CHANGELOG.md

```diff
@@ -2,6 +2,79 @@
 
 ## Unreleased
 
+## v0.5.0 Beta
+
+## 🛑 Breaking changes 🛑
+
+- Enable "new-metrics" by default (#1148)
+- Remove `export_resource_labels` flag from Zipkin exporter (#1163)
+- Add config settings for Http Client, change Zipkin to use it. (#1186)
+- Update OTLP to v0.4.0 (#1142)
```

Comment:
This is major breaking change. It makes Collector incompatible with any other sender or receiver of OTLP protocol of different version. This needs big red warning.